pyproject.toml

```diff
@@ -1,8 +1,73 @@
+[project]
+name = "airbase"
+dynamic = ["version"]
+requires-python = ">=3.8,<4"
+description = "An easy downloader for the AirBase air quality data."
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [{name="John Paton", email="john@johnpaton.net"}]
+maintainers = [{name="John Paton", email="john@johnpaton.net"}, {name="Alvaro Valdebenito"}]
```

Comment:
let me know if you want to be here and/or have your email here @avaldebe

Reply:
@JohnPaton sorry, I did not see this comment until now.
My work email is `alvarov@met.no`

Reply:
All good! I'll update it.